package.json

```diff
@@ -12,7 +12,10 @@
     "hof": "^12.0.0",
     "hof-behaviour-summary-page": "^2.0.0",
     "hof-build": "^1.3.1",
-    "hof-theme-govuk": "^2.0.2"
+    "hof-form-controller": "^5.0.0",
+    "hof-model": "^3.1.1",
+    "hof-theme-govuk": "^2.0.2",
+    "lodash": "^4.17.4"
```

Comment:
The only place I can see that uses `lodash` is to do a `_.map`, which we could just as easily use native `Array#map` for and not need the extra dependency.